rice-middleware/impl/src/main/java/org/kuali/rice/kew/routelog/web/RouteLogAction.java

```diff
@@ -155,7 +157,12 @@ private ActionRequestValue switchActionRequestPositionIfPrimaryDelegatePresent(
     	 */
     	
     	if (!actionRequest.isRoleRequest()) {
-    		List<ActionRequestValue> primaryDelegateRequests = actionRequest.getPrimaryDelegateRequests();
+            /**
+             * KULRICE-14183 : Detaching the action request object so that re-ordering of action requests for primary delegates does not get incorrectly persisted
+             */
+            EntityManager entityManager = GlobalResourceLoader.getService("rice.kew.entityManager");
```

Comment:
Good catch.  We missed the same in NoteAction.java (KULRICE-13319). 

I'm wondering if we should have a reference to EntityManager at all in this class. @ssaripel hold on with corrections until we advise you on this.  Thanks.